templates/an-cash-dao/contracts/BaseTemplate.sol

```diff
@@ -0,0 +1,359 @@
+    }
+
+    function _createVaultPermissions(ACL _acl, Vault _vault, address _grantee, address _manager) internal {
+        _acl.createPermission(_grantee, _vault, _vault.TRANSFER_ROLE(), _manager);
+    }
+
+    /* DISPUTABLE VOTING */
+
+    function _installDisputableVotingApp(Kernel _dao, MiniMeToken _token, uint64[7] memory _votingSettings) internal returns (DisputableVoting) {
+        return _installDisputableVotingApp(_dao, _token, _votingSettings[0], _votingSettings[1], _votingSettings[2], _votingSettings[3], _votingSettings[4], _votingSettings[5], _votingSettings[6]);
+    }
+
+    function _installDisputableVotingApp(
+        Kernel _dao,
+        MiniMeToken _token,
+        uint64 _duration,
+        uint64 _support,
+        uint64 _acceptance,
+        uint64 _delegatedVotingPeriod,
+        uint64 _quietEndingPeriod,
+        uint64 _quietEndingExtension,
+        uint64 _executionDelay
+    )
+        internal returns (DisputableVoting)
+    {
+        bytes memory initializeData = abi.encodeWithSelector(DisputableVoting(0).initialize.selector, _token, _duration, _support, _acceptance, _delegatedVotingPeriod, _quietEndingPeriod, _quietEndingExtension, _executionDelay);
+        return DisputableVoting(_installNonDefaultApp(_dao, DISPUTABLE_VOTING_APP_ID, initializeData));
+    }
+
+    function _createDisputableVotingPermissions(
+        ACL _acl,
+        DisputableVoting _voting,
+        address _grantee,
+        address _manager
+    )
+        internal
+    {
+        _acl.createPermission(ANY_ENTITY, _voting, _voting.CHALLENGE_ROLE(), _manager);
+
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_VOTE_TIME_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_SUPPORT_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_QUORUM_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_DELEGATED_VOTING_PERIOD_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_QUIET_ENDING_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_EXECUTION_DELAY_ROLE(), _manager);
+    }
+
+    /* FINANCE */
+
+    function _installFinanceApp(Kernel _dao, Vault _vault, uint64 _periodDuration) internal returns (Finance) {
+        bytes memory initializeData = abi.encodeWithSelector(Finance(0).initialize.selector, _vault, _periodDuration);
+        return Finance(_installNonDefaultApp(_dao, FINANCE_APP_ID, initializeData));
+    }
+
+    function _createFinancePermissions(ACL _acl, Finance _finance, address _grantee, address _manager) internal {
+        _acl.createPermission(_grantee, _finance, _finance.EXECUTE_PAYMENTS_ROLE(), _manager);
+        _acl.createPermission(_grantee, _finance, _finance.MANAGE_PAYMENTS_ROLE(), _manager);
+    }
+
+    function _createFinanceCreatePaymentsPermission(ACL _acl, Finance _finance, address _grantee, address _manager) internal {
```

Comment:
Not sure, but perhaps we can remove these functions around `CREATE_PAYMENTS_ROLE` to put them into `_createFinancePermissions()`, now that we don't need to assign it to the Payroll app?